### PR TITLE
Events functions

### PR DESCRIPTION
**Pull request by MathieuAndrade**

### CODE REVIEW

File: api/core/event/index.js [hunk 1]
```diff
@@ -2,5 +2,6 @@
 module.exports.create = require('./event.create.js');
 module.exports.command = require('./event.command.js');
 module.exports.get = require('./event.get.js');
+module.exports.getByEventtype = require('./event.getByEventtype.js');
```
Comment (Pierre-Gilles) [inline api/core/event/index.js]: Juste one feedback, can you call your function `getByEventType` ? camelcase is more beautiful :)

I know it's not the case everywhere in Gladys, but the goal is to be camelcase everywhere in the JS code

**Comment by Pierre-Gilles** [comment]
Be careful, you didn't changed everywhere the name of the function! (documentation for example)

+ you commited changes in other files that are not related to this release (layout.ejs)

**Comment by MathieuAndrade** [comment]
Oh sorry ! I was too fast ^^

**Comment by Pierre-Gilles** [comment]
Waiting for your changes so :p

**Comment by MathieuAndrade** [comment]
It's good ! ^^

**Pull request merged**